### ISSUE TITLE
[BUG] open_point_in_time uses request params, but body is needed. 

### ISSUE THREAD

**Issue by arittner** [opensearch-project/opensearch-py#165]
**What is the bug?**
A call to 
`resp = client.open_point_in_time(index=index_name, keep_alive='10m')`
is not working. OpenSearch expects a body.

Exception:
~~~
Traceback (most recent call last):
  File "C:\DevTools\Core\Python39\lib\site-packages\opensearchpy\transport.py", line 395, in perform_request
    if retry:
  File "C:\DevTools\Core\Python39\lib\site-packages\opensearchpy\transport.py", line 368, in perform_request
    status, headers_response, data = connection.perform_request(
  File "C:\DevTools\Core\Python39\lib\site-packages\opensearchpy\connection\http_urllib3.py", line 275, in perform_request
    self._raise_error(
  File "C:\DevTools\Core\Python39\lib\site-packages\opensearchpy\connection\base.py", line 300, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(
opensearchpy.exceptions.RequestError: RequestError(400, 'parse_exception', 'request body is required')
~~~

**How can one reproduce the bug?**
Try to open a PIT with open_point_in_time (see above)

**What is the expected behavior?**
Successful call to create a PIT response. 

**What is your host/environment?**
 - Docker Desktop (Windows 11)
 - Opensearch-py 1.1.0
 - OpenSearch 1-3-1

**Comment by wbeckler** [comment]
This is a good catch. Would you be up for submitting a fix?

**Comment by harshavamsi** [comment]
@saimedhi can you take a look at this?

**Comment by harshavamsi** [comment]
@arittner `open_point_in_time` is not supported on `opensearch-py` currently. We anticipate this will be released in `opensearch-py` version `2.1` and will only be supported with `OpenSearch 2.4`. Here's the documentation[1] on this API, it has now been refactored to `create_point_in_time`.

[1] https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md#point-in-time-api
